api/src/main/java/org/apache/gravitino/SupportsCatalogs.java

```diff
@@ -78,12 +78,14 @@ default boolean catalogExists(String catalogName) {
    * Create a catalog with specified catalog name, type, provider, comment, and properties.
    *
    * <p>The parameter "provider" is a short name of the catalog, used to tell Gravitino which
-   * catalog should be created. The short name should be the same as the {@link CatalogProvider}
-   * interface provided.
+   * catalog should be created. The short name:
+   *
+   * <p>1) should be the same as the {@link CatalogProvider} interface provided. 2) can be "null" if
+   * the created catalog is a built-in catalog, like model catalog.
```

Comment:
I'm very puzzle about the concept of `built-in` catalog, can you explain it more clearly.

Reply:
And, so the provider of all the built-in catalog is null?

Reply:
From the user level, the "provider" value can be null, user doesn't need to provide the "provider" for those catalogs. 

From the Gravitino kernel, it will treat the catalog "type" name as the "provider" name, and use this name to load in the catalog.